Selecting the GS/FNC1 separator for DataMatrix

POKKAHOH opened the pull request:
In [GS1 General Specifications Release 21.0.1, Ratified, Jan 2021](https://www.gs1.org/docs/barcodes/GS1_General_Specifications.pdf)
p. 488 item "7.8.3 The separator character and its value" reads
"... In GS1 DataMatrix and GS1 DotCode symbology: The Function 1 Symbol Character (FNC1) or the
control character <GS> SHALL be the separator character. ..."

Some changes should be made to DataMatrix.java

mabsofxanadu commented:
same changes i requested a while back:
https://github.com/woo-j/OkapiBarcode/pull/62

but it was closed saying it is not needed anymore.
i am being asked about this for a new project, previously i was using my forked version with the changes, but i would really like this changes to be integrated in the standard okapi.

so a +1 for me for this change.

gredler commented:
It sounds like this feature makes sense. A few questions and comments:
1. Why the change to .travis.yml?
2. The booleans / setter / getter should probably be named useGS (not useGS1).
3. Can you add 3 unit tests that cover the 3 new useGS checks? See the SymbolTest javadoc for more info on tests.

POKKAHOH commented:
Thanks for the comments.
Accepted.
1. I'm newbie to java and I'm not sure about my skills, so I turned off an incomprehensible tool to go through the assembly.
2. I agree. Fixed.
3. Not sure if I will do it correctly and quickly. (see item 1)
4. Fixed.

gredler commented:
Thanks for the cleanup. Have you had a chance to create some test cases to exercise the 3 new branches?

gredler commented:
Thanks for your submission -- I've committed a separate fix with the necessary test cases, so you should be able to use this feature in the next version. For more info, see commit 2e0765f0eb1cd2549fdd01267984db3e9c3fe96a.
